lib/src/main/kotlin/com/swmansion/starknet/extensions/json.kt

```diff
@@ -0,0 +1,9 @@
+package com.swmansion.starknet.extensions
+
+import com.swmansion.starknet.data.types.Felt
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonObjectBuilder
+import kotlinx.serialization.json.JsonPrimitive
+
+fun JsonObjectBuilder.put(key: String, value: Felt?): JsonElement? =
+    put(key, JsonPrimitive(value?.hexString() ?: Felt.ZERO.hexString()))
```

Comment:
Do we want to use the default value here?

Reply:
Perhaps we could change this method to only accept non null Felts?